scripts/update-sitemap.ts

```diff
@@ -0,0 +1,240 @@
+    events.push(...data);
+
+    url = result.meta.next;
+  }
+
+  return events;
+};
+
+/**
+ * Generate sitemap in xml format for events in selected language
+ * @param {object[]} events
+ * @param {string} language
+ */
+const generateEventSitemap = async (
+  events: SimpleEvent[],
+  language: string
+) => {
+  const data = {
+    _declaration: { _attributes: { encoding: "utf-8", version: "1.0" } },
+    urlset: {
+      _attributes: { xmlns: "http://www.sitemaps.org/schemas/sitemap/0.9" },
+      url: [
+        ...events.map(event => ({
+          lastmod: formatDate(event.last_modified_time),
+          loc: `${host}/${language}/event/${event.id}`
+        }))
+      ]
+    }
+  };
+
+  await writeXMLFile(`${pathToSitemaps}/sitemap_events_${language}.xml`, data);
+};
+
+/**
+ * Generate sitemap in xml format for events in all languages
+ */
+const generateEventSitemaps = async () => {
+  await Promise.all(
+    languages.map(async lang => {
+      try {
+        const events = await getEvents(lang);
+        // eslint-disable-next-line no-console
+        await generateEventSitemap(events, lang);
+      } catch (err) {
+        throw err;
+      }
+    })
+  );
+};
+
+/**
+ * Fetch collections from cms
+ * @return {object[]}
+ */
+const getCollections = async (): Promise<SimpleCollection[]> => {
+  const collections: SimpleCollection[] = [];
+  const url = `${cmsUrl}/collections`;
+
+  const start = new Date();
+  const res = await fetch(url);
+  const result = await res.json();
+
+  if (res.status !== 200) {
+    throw Error(
+      `Could not fetch collections data from url ${url}: ${res.status} ${res.statusText}`
+    );
+  }
+
+  const end = new Date();
+  // eslint-disable-next-line no-console
+  console.log(`GET: ${url} (${end.getTime() - start.getTime()}ms)`);
+
+  collections.push(...result);
+
+  return collections;
+};
+
+/**
+ * Generate sitemap in xml format for collections in selected language
+ * @param {object[]} collections
+ * @param {string} language
+ */
+const generateCollectionSitemap = async (
+  events: SimpleCollection[],
+  language: string
+) => {
+  const data = {
+    _declaration: { _attributes: { encoding: "utf-8", version: "1.0" } },
+    urlset: {
+      _attributes: { xmlns: "http://www.sitemaps.org/schemas/sitemap/0.9" },
+      url: [
+        ...events.map(event => ({
+          lastmod: formatDate(event.last_published_at),
+          loc: `${host}/${language}/event/${event.id}`
+        }))
+      ]
+    }
+  };
+
+  await writeXMLFile(
+    `${pathToSitemaps}/sitemap_collections_${language}.xml`,
+    data
+  );
+};
+
+/**
+ * Generate sitemaps in xml format for collections in all languages
+ */
+const generateCollectionSitemaps = async () => {
+  const collections = await getCollections();
+  await Promise.all(
+    languages.map(async lang => {
+      try {
+        await generateCollectionSitemap(collections, lang);
+      } catch (err) {
+        throw err;
+      }
+    })
+  );
```

Comment:
```suggestion
 Promise.all(languages.map(lang => await generateCollectionSitemap(collections, lang)));
```
I have same comments as above; I think this would do the trick.